src/pages/tezos/claim-transaction.js

```diff
@@ -409,17 +410,39 @@ function Transaction({ location }) {
                   )}
                 </div>
                 <div className="grid mx-auto justify-center mt-6">
-                  <Button
+                  <button
                     onClick={() => {
                       if (claimButtonDisabled) return;
+                      setClaimButtonDisabledStatus(true);
                       mintNFT();
                     }}
                     size="lg"
-                    type="primary"
+                    className={`${
```

Comment:
In case the balance is empty it's showing lighter blue color.
![Screenshot 2021-03-24 at 1 56 04 PM](https://user-images.githubusercontent.com/16613455/112278496-e76ce580-8ca8-11eb-98f7-c944e9190bd0.png)

But, in the marketplace transaction view. It's showing darker blue color.
![Screenshot 2021-03-24 at 1 55 23 PM](https://user-images.githubusercontent.com/16613455/112278596-053a4a80-8ca9-11eb-8823-71d447e9be58.png)

The discrepancy here needs to be fixed.